platform/posix/utest/reconnect_utest.c

```diff
@@ -74,6 +81,7 @@ void test_Transport_ReconnectBackoffAndSleep_Succeeds( void )
     bool retriesArePending;
     int32_t expectedNextJitterMax = INITIAL_RECONNECT_BACKOFF_SECONDS;
 
+    rand_ExpectAndReturn( JITTER );
```

Comment:
This call is superfluous for the `Transport_ReconnectParamsReset` function

Reply:
So do you want me to mock rand or not?

Reply:
Nevermind, I forgot that `Transport_ReconnectParamsReset` calls `rand`